### ISSUE TITLE
Make a survey for Scholia users

### ISSUE THREAD

**Issue by fnielsen** [WDscholia/scholia#2548]
**Is your feature request related to a problem? Please describe.**
We would like to have a survey of Scholia users

**Describe the solution you'd like**
* On Scholia (all pages?) we would like to have a link to the survey
* Perhaps Google Forms? Or is there other Wikimedia-oriented survey. Lime survey?

**Comment by fnielsen** [comment]
"Wikimedia Foundation has a LimeSurvey instance available to staff members" - https://meta.wikimedia.org/wiki/LimeSurvey/

**Comment by Daniel-Mietchen** [comment]
I have set up https://scholia.limesurvey.net  and am playing with the options there.

**Comment by Daniel-Mietchen** [comment]
I set up a test survey at https://scholia.limesurvey.net/765572 .

**Comment by Daniel-Mietchen** [comment]
I think the free version of LimeSurvey cloud is not an option, as it is limited to 25 responses:

![grafik](https://github.com/user-attachments/assets/ca58f11d-20a8-4be4-b326-d8d6992d7217)

They offer several paid plans at https://www.limesurvey.org/pricing : 

![grafik](https://github.com/user-attachments/assets/ba20f6ed-2170-4118-a3d8-5635f8826aaf)

I asked at the Wikimedia Hackathon Telegram channel whether anyone knows of an instance on Wikimedia servers.

Shall we host our own instance?

**Comment by Daniel-Mietchen** [comment]
A bot in the Telegram channel pointed me to 
https://meta.wikimedia.org/wiki/Survey_Support_Desk
and
https://meta.wikimedia.org/wiki/LimeSurvey . The latter (already mentioned by Finn above) has a table
![grafik](https://github.com/user-attachments/assets/f2219fb4-bdaf-4d04-8b67-a64eef56b1eb)
that mentions https://survey.wikisp.org/ (again, as pointed out by Finn).
So maybe we have to set up a Phabricator ticket for this. Will look into that.

**Comment by fnielsen** [comment]
Do we know where [limesurvey.net](https://scholia.limesurvey.net/)-based LimeSurvey data goes? In terms of GDPR? And who is Wikimedia Small Projects?

**Comment by fnielsen** [comment]
Can you move an a LimeSurvey questionaire from one LimeSurvey instance to another?

**Comment by fnielsen** [comment]
Possible questions:
* How did you come here?
 * Typed home address
 * From Wikimedia Commons
 * From a search engine
* Have you  heard of Scholia (or do you know what it is?)
* What do you use it for?
* What is missing in Scholia? (data: papers, citation, authors, topics) (features: I would like to...)  
* How much do you use Scholia?

**Comment by Daniel-Mietchen** [comment]
A request for this survey to be ported to LimeSurvey has been drafted, and it includes draft questions: 
https://meta.wikimedia.org/wiki/Wikimedia_Italia/LimeSurvey/Requests/Scholia_user_survey_2024 .

**Comment by fnielsen** [comment]
I have added to https://meta.wikimedia.org/wiki/Wikimedia_Italia/LimeSurvey/Requests/Scholia_user_survey_2024#Discussion

"""Other questions could be "How would you rate Scholia?" (optional, single choice on a scale). "What do you like about Scholia?" (optional, free text)."""

**Comment by Daniel-Mietchen** [comment]
The survey is now live at https://survey.wikimedia.it/index.php/179555 . Let's do some final checks, and then we should link it from Scholia for some time.

**Comment by fnielsen** [comment]
Should I do a test answer?

**Comment by Daniel-Mietchen** [comment]
@fnielsen Yes, you could do that, and if everything looks fine, please add the banner.

**Comment by fnielsen** [comment]
"I export the content for use in my own tools". How is that relevant for Scholia: through RSS or the WDQS links?

"What is your Wikimedia username?". This should/could be optional.

**Comment by egonw** [comment]
What is the status of the survey? Are we done testing? Will we announce it this week?

**Comment by bluerasberry** [comment]
I made two changes:
_"What is your Wikimedia username?". This should/could be optional._
It is now - "Mandatory: 	No"
_"I export the content for use in my own tools". How is that relevant for Scholia: through RSS or the WDQS links?_
I changed to "I reuse the content elsewhere"

We have a few test surveys completed. It is working and can be shared now.

**Comment by egonw** [comment]
> can be shared now.

@bluerasberry, as in, we will start sharing today?

**Comment by fnielsen** [comment]
Now runs: https://scholia.toolforge.org/

**Comment by Daniel-Mietchen** [comment]
Screenshots: 

![Image](https://github.com/user-attachments/assets/190b069a-ceb5-4a42-8873-5949172a9604)

![Image](https://github.com/user-attachments/assets/ce0d976d-805f-4410-bfd6-2c1986568218)

**Comment by lubianat** [comment]
I made some plots on the results of the User Survey and made them available through Zenodo and Commons: 

https://zenodo.org/records/17491523

https://upload.wikimedia.org/wikipedia/commons/2/24/Scholia_User_Survey_2025.pdf
